test/modules/fundingManager/bondingCurve/FM_BC_Bancor_Redeeming_VirtualSupply_v1.t.sol

```diff
@@ -59,10 +59,10 @@ contract FM_BC_Bancor_Redeeming_VirtualSupplyV1Test is ModuleTest {
     uint8 internal constant DECIMALS = 18;
     uint internal constant MAX_SUPPLY = type(uint).max;
 
-    uint internal constant INITIAL_ISSUANCE_SUPPLY = 1;
-    uint internal constant INITIAL_COLLATERAL_SUPPLY = 3;
-    uint32 internal constant RESERVE_RATIO_FOR_BUYING = 333_333;
-    uint32 internal constant RESERVE_RATIO_FOR_SELLING = 333_333;
+    uint internal constant INITIAL_ISSUANCE_SUPPLY = 195_642_169e16;
+    uint internal constant INITIAL_COLLATERAL_SUPPLY = 39_097_931e16;
+    uint32 internal constant RESERVE_RATIO_FOR_BUYING = 199_800;
+    uint32 internal constant RESERVE_RATIO_FOR_SELLING = 199_800;
```

Comment:
Are these the new values that produce better initial pricing values? Is this supposed to be in this PR?

Reply:
They are values from the TEC launch; "real world values"